episodes/github.md

```diff
@@ -1,156 +1,188 @@
 ---
-title: "Uploading a coding project to GitHub"
+title: "Pre-workshop assignment: Uploading a coding project to GitHub"
 teaching: 5
 exercises: 45
 ---
 
-:::::::::::::::::::::::::::::::::::::: questions 
+::: questions
+-   How do I share my changes with others on the web?
+:::
 
-- How do I share my changes with others on the web?
+::: objectives
+-   Create a repository on GitHub
+-   Push to or pull from a remote repository.
+:::
 
-::::::::::::::::::::::::::::::::::::::::::::::::
+## Creating a GitHub repository
 
-::::::::::::::::::::::::::::::::::::: objectives
+You are going to add your existing project to GitHub
```

Comment:
Expand further, make it clear what the goal is of this assignment. Explain a little bit why we put our code on GitHub. Add instructions for if your project is already on GitHub.